perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -3983,19 +4449,23 @@ public void checkAttributeDependencies(PerunSession sess, RichAttribute richAttr
 				List<Vo> vosFromUser = new ArrayList<Vo>();
 				List<Member> membersFromUser = getPerunBl().getMembersManagerBl().getMembersByUser(sess, user);
 				List<Resource> resourcesFromUser = new ArrayList<Resource>();
-				for(Member memberElement: membersFromUser) {
+				for (Member memberElement : membersFromUser) {
```

Comment:
If you want to change formatting, i prefer to do it in another commit. If you need to do it in this one, please add some information to commit message. Maybe your IDE do automatic formatting, be carefull about that. It's harder to check this code if there are also formatting changes.